hedera-mirror-importer/src/main/java/com/hedera/mirror/importer/MirrorProperties.java

```diff
@@ -54,7 +53,7 @@ public class MirrorProperties {
     private boolean leaderElection = false;
 
     @NotNull
-    private Map<String, MigrationProperties> migration = Collections.emptyMap();
+    private CaseInsensitiveMap<String, MigrationProperties> migration = new CaseInsensitiveMap<>();
```

Comment:
this and the other place can be just
```suggestion
    private Map<String, MigrationProperties> migration = new CaseInsensitiveMap<>();
```